subo/command/build.go

```diff
@@ -48,6 +48,14 @@ func BuildCmd() *cobra.Command {
 			shouldBundle := !noBundle && !bctx.CwdIsRunnable
 
 			useNative, _ := cmd.Flags().GetBool("native")
+			makeStr, _ := cmd.Flags().GetString("make")
+
+			util.LogStart(fmt.Sprintf("make %s", makeStr))
+			_, _, err = util.Run(fmt.Sprintf("make %s", makeStr))
+
```

Comment:
Please remove this blank line and add a blank line after the `if err != nil` closing brace.